pkg/render/common/elasticsearch/decorator.go

```diff
@@ -28,6 +28,8 @@ import (
 const (
 	elasticsearchSecretsAnnotation   = "hash.operator.tigera.io/elasticsearch-secrets"
 	elasticsearchConfigMapAnnotation = "hash.operator.tigera.io/elasticsearch-configmap"
+	TigeraCertBundleMountPathWindows = "c:/etc/fluentd/elastic/tigera-ca-bundle.crt"
+	TigeraCertBundleMountPath        = "/etc/fluentd/elastic/tigera-ca-bundle.crt"
```

Comment:
Please revert the changes in this file. this is not the right way to change this.

Reply:
Done